support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -120,6 +122,7 @@ function init({
 	const urlParticipations = getParticipationsFromUrl();
 	const serverSideParticipations = getServerSideParticipations();
 	return {
+		...sessionParticipations,
```

Comment:
`sessionParticipations` could be `undefined`, can you use a spread operator on `undefined`? I would've expected TS to complain here 🤔

Reply:
Good question. It appears you _can_ spread on undefined - and here `serverSideParticipations` and `urlParticipations` are also potentially undefined.
I guess we could be a bit more deliberate about it with `...(sessionParticipations ?? {})`